src/components/Modal/Modal.tsx

```diff
@@ -49,6 +49,7 @@ interface Props {
    * Styilng through inline CSSProperties
    */
   style?: CSSProperties
+  [x: string]: any
```

Comment:
I don't think that having this component accept any prop is a good idea for the following reasons: 

1. It creates an undefined interface which leads to developer confusion
2. It does not allow the component to be documented
3. In this case, it leaks the implementation details since we are using it to essentially expose the props of the underlying modal implementation. Making sure that the modal implementation could be changed without breaking the public interface is important for this library.

We should refactor this to expose the props that we want to use for customization with proper names for the HospitalRun context.

Reply:
I will gladly change for better documentations, but I'm wondering why we would need to if the `hospitalRun/components` is just a wrapper for `react-bootstrap` which has good documentation?

Now I understand if at some point another new better `component library` comes around that then you would like to change to.  Having everything just passed down will require a TON! of refactoring on the actual site vs just in the component.

Well I just convinced myself into not allowing the props to be passed down.  I'll correct this request.